oval-schemas/oval-definitions-schema.xsd

```diff
@@ -1103,6 +1114,33 @@
             -->
         </xsd:restriction>
     </xsd:simpleType>
+    <xsd:simpleType name="SortEnumeration">
+        <xsd:annotation>
+            <xsd:documentation>The SortEnumeration simple type defines basic sorting operations.  Currently 'none', 'alphabetical' and 'alphanumerical' are defined..</xsd:documentation>
+        </xsd:annotation>
+        <xsd:restriction base="xsd:string">
+            <xsd:enumeration value="none">
```

Comment:
I wouldn't call this "none", I would call this "document_order".  After all, two interpreters using the same system-characteristics file as input should both generate identical results.

Reply:
I wasn't entirely sure what your "document_order" meant, as long as we explain it as the order of the items/values in  OVAL system characteristics, I would agree.

Reply:
Document order means... this is an XML document.  The order is the order in which the associated elements appear in the document.

Reply:
In my brain, this all happens in memory structures, long before we end up writing it to XML, so that was my disconnect.